libs/vkd3d/vkd3d_private.h

```diff
@@ -113,6 +113,11 @@ struct vkd3d_vulkan_info
     bool KHR_timeline_semaphore;
     bool KHR_shader_float16_int8;
     bool KHR_shader_subgroup_extended_types;
+    bool KHR_ray_tracing_pipeline;
+    bool KHR_acceleration_structure;
+    bool KHR_deferred_host_operations;
+    bool KHR_spirv_1_4;
```

Comment:
SPIR-V 1.4 is required for the KHR extension, and shader_float_controls is a dependency of 1.4.